app-frontend/webpack.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = function (options = {}) {
 
   const APP_DIR = path.resolve(__dirname, "app");
 
-  console.log(`
+  console.error(`
```

Comment:
Why `error`?

Reply:
Cause `log` breaks default flow of https://github.com/webpack/analyse
It's easy work around, to not manually change stats.json